css/src/mixins/checkbox.scss

```diff
@@ -0,0 +1,11 @@
+@mixin checkmark {
```

Comment:
I think that this mixin should live in the file with checkbox. The mixins folder is a better place for mixins that need to be globally available, such as `visually-hidden`.

Reply:
Moved to the checkbox file